xwiki-platform-distribution/xwiki-platform-distribution-war-legacydependencies/pom.xml

```diff
@@ -62,6 +62,11 @@
                     <exclude>org.xwiki.platform:xwiki-platform-rendering-macro-include:*:jar:*</exclude>
                     <exclude>org.xwiki.platform:xwiki-platform-rendering-wikimacro-api:*:jar:*</exclude>
                     <exclude>org.xwiki.platform:xwiki-platform-rendering-wikimacro-store:*:jar:*</exclude>
+                    <exclude>org.xwiki.platform:xwiki-platform-notifications-api:*:jar:*</exclude>
+                    <exclude>org.xwiki.platform:xwiki-platform-notifications-filters-api:*:jar:*</exclude>
+                    <exclude>org.xwiki.platform:xwiki-platform-notifications-notifiers-api:*:jar:*</exclude>
+                    <exclude>org.xwiki.platform:xwiki-platform-notifications-notifiers-default:*:jar:*</exclude>
+                    <exclude>org.xwiki.platform:xwiki-platform-notifications-sources:*:jar:*</exclude>
```

Comment:
I don't see any aspect in xwiki-platform-legacy-notifications-sources. Keep in mind that the point of that list is only to indicate which JAR are **replaced** by legacy JAR and this only happen when the legacy module repackage the non legacy one.

Reply:
Ok so I indeed added too many exclusions here. I'll fix it.